src/openrct2/ride/Vehicle.cpp

```diff
@@ -6945,21 +6945,21 @@ static int32_t vehicle_get_swing_amount(Vehicle* vehicle)
  *
  *  rct2: 0x006D6776
  */
-static void vehicle_update_swinging_car(Vehicle* vehicle)
+void Vehicle::UpdateSwingingCar()
 {
     int32_t dword_F64E08 = abs(_vehicleVelocityF64E08);
-    vehicle->var_4E += (-vehicle->swinging_car_var_0) >> 6;
-    int32_t swingAmount = vehicle_get_swing_amount(vehicle);
+    var_4E += (-swinging_car_var_0) >> 6;
+    int32_t swingAmount = GetSwingAmount();
     if (swingAmount < 0)
     {
-        vehicle->var_4E -= dword_F64E08 >> (-swingAmount);
+        var_4E -= dword_F64E08 >> (-swingAmount);
     }
     else if (swingAmount > 0)
     {
-        vehicle->var_4E += dword_F64E08 >> swingAmount;
+        var_4E += dword_F64E08 >> swingAmount;
```

Comment:
`var_4E` is probably the swinging boat position while `dword_F64E08` seems to be the increment base?